DatePicker/tests/DatePicker-specs.js

```diff
@@ -161,4 +191,10 @@ describe('DatePicker', () => {
 
 		expect(header).toBeNull();
 	});
+
+	test('"dateToLocaleString" method should return "null" for an "undefined" date', () => {
```

Comment:
why was this test needed?

Reply:
I would like to keep this test because we export that function to be used on it's own. As such we need to ensure it's behaviour for anomalous props.